js/src/telegram/messages.ts

```
@@ -401,15 +402,16 @@ export const listenForEvents = async (
 		const user = await getUserInfo(client, userId);
-
 		if (!user) {
 			return;
 		}
-
 		switch (event.className) {
+			case "UpdateUserTyping":
+				updateUserTyping(user)
+				break
 			case 'UpdateShortMessage':
 				const config = getConfig('notifications');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)